terasoluna-gfw-parent/pom.xml

```diff
@@ -311,6 +313,19 @@
             </dependency>
             <!-- == End Spring IO Platform == -->
 
+            <!-- == Start Tomcat == -->
+            <dependency>
+                <groupId>org.apache.tomcat</groupId>
+                <artifactId>tomcat-jsp-api</artifactId>
+                <version>${tomcat8.version}</version>
```

Comment:
- [ ] Why don't you use ``javax.servlet.jsp-api`` defined in Spring IO Platform. This does not need to define in ``<dependencyManagement/>``.

http://docs.spring.io/platform/docs/current/reference/htmlsingle/